vocab/credentials/v2/vocabulary.yml

```
@@ -153,6 +161,12 @@ property:
     status: reserved
     range: cred:RenderMethod
 
+  - id: relatedResource
+    label: Related resource
+    domain: cred:VerifiableCredential
+    range: IRI
+    defined_by: https://w3c-ccg.github.io/vc-render-method/#defn-relatedResource
```

Copy & paste typo.  I think this is the correct link:
```suggestion
    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#defn-relatedResource
```